app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt

```diff
@@ -93,9 +99,62 @@ class TopicInfoFragmentTest {
     }
   }
 
+  @Test
+  fun testTopicInfoFragment_loadFragment_checkDefaultTopicDescriptionLines_fiveLinesVisible() {
+    launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
+      onView(withId(R.id.topic_description_text_view)).check(matches(maxLines(/* lineCount= */ 5)))
+    }
+  }
+
+  @Test
+  fun testTopicInfoFragment_loadFragment_seeMoreIsVisible() {
```

Comment:
This testcase is failing.
<img width="1635" alt="Screenshot 2020-03-30 at 11 49 51 AM" src="https://user-images.githubusercontent.com/16301028/77881447-a6e8b080-727c-11ea-8de9-ecfcad5219f0.png">